numpy/random/src/common/LICENSE.md

```diff
@@ -0,0 +1,29 @@
+ISO C9x  compliant stdint.h for Microsoft Visual Studio
```

Comment:
This file and references to it can be removed.  It is a Python2.7/windows hack only AFAIK.

Reply:
This is done in a PR